client/src/main/java/com/paypal/selion/reports/reporter/runtimereport/JsonRuntimeReporterHelper.java

```diff
@@ -83,7 +83,8 @@
     public JsonRuntimeReporterHelper() {
         try {
             File workingDir = new File(Config.getConfigProperty(ConfigProperty.WORK_DIR));
-            workingDir.mkdirs();
+            boolean isCreated = workingDir.mkdirs();
+            logger.log(Level.FINER,"Is working directory creation successful:"+ isCreated);
```

Comment:
Please reword this message and add fix the formatting.

I suggest;

```java
logger.log(Level.FINER, "Working directory created successfully: " + isCreated);
```

That said, we don't seem to be doing anything if it was not successfully created? Should we be? If not, then why bother logging the message?